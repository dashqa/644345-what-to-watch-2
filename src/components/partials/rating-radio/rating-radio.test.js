import React from 'react';
import renderer from 'react-test-renderer';
import RatingRadio from './rating-radio';

const props = {
  id: `my-input-1`,
  value: 1,
  checkedFlag: 1,
  onChange: () => {}
};

it(`Rating radio poster correctly renders after relaunch`, () => {
  const tree = renderer.create(<RatingRadio {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
