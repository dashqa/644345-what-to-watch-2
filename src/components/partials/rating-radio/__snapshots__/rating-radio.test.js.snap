// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating radio poster correctly renders after relaunch 1`] = `
Array [
  <input
    checked={true}
    className="rating__input"
    id="my-input-1"
    name="rating"
    onChange={[Function]}
    type="radio"
    value={1}
  />,
  <label
    className="rating__label"
    htmlFor="my-input-1"
  >
    Rating 1
  </label>,
]
`;
